continuous-pipe.yml

```diff
@@ -128,9 +128,12 @@ tasks:
         magento1_nginx:
           image: quay.io/continuouspipe/magento1-nginx-php5.6
           tag: latest
-        magento2_nginx:
+        magento2_nginx_php7:
           image: quay.io/continuouspipe/magento2-nginx-php7
           tag: latest
+        magento2_nginx_php71:
+          image: quay.io/continuouspipe/magento2-nginx-php7.1
+          tag: latest
```

Comment:
I'll rename these magento2_php{version}_{web server} for consistency in `doc build`